src/Microsoft.Azure.WebJobs.Host/Scale/ScaleMonitorDescriptor.cs

```diff
@@ -13,6 +13,11 @@ public ScaleMonitorDescriptor(string id)
             Id = id;
         }
 
+        public ScaleMonitorDescriptor(string id, string functionId) : this(id)
```

Comment:
Since our goal going forward is for people to use this new constructor, I recommend marking the other one [Obsolete] to start people migrating

Reply:
FYI this is potentially a breaking change for consumers of this library. If they have WarnAsError configured as part of their build, it will break them.